### ISSUE TITLE
"Failed to setupup dukaone device"

### ISSUE THREAD

**Issue by Aephir** [dingusdk/ha-dukaone#9]
I get this error whenever I start HA.

They seem to work, though, so I'm not sure if it's just a timing thing?

```
This error originated from a custom integration.

Logger: custom_components.dukaone.sensor
Source: custom_components/dukaone/sensor.py:32
Integration: Duka One (documentation)
First occurred: 14:45:41 (1 occurrences)
Last logged: 14:45:41

Failed to setupup dukaone device
```

And since I'm at it, I think there's a typo in line 32 of `sensor.py`, I think

```
_LOGGER.error("Failed to setupup dukaone device")
```
should probably be
```
_LOGGER.error("Failed to set up dukaone device")
```

:)

**Comment by dingusdk** [comment]
When you get this error the humidity sensor is not setup in HA. It is a timeout if the duka one device has not send any humidity data for 10 sec. The fan may stil work fine in HA.

What model do you have? Maybe there are some without humidity sensor.

**Comment by Aephir** [comment]
[Duka one S6 Plus Wifi](https://dukaventilation.dk/produkter/1-rums-ventilationsloesninger/duka-one-s6-plus-wifi).

The [documentation](https://dukaventilation.dk/Admin/Public/DWSDownload.aspx?File=%2fFiles%2fDukaVentilation%2fPDF%2fBruger-+og+montagemanualer%2fDUKA+One+manualer%2fBrugermanual+DUKA+One+Pro+S4_S6+Plus+2023.pdf) is a bit difficult t assess, page 11 states that an external moisture sensor can be attached, page 16 talks about the built in moisture sensor, and page 22 talks about "either the moisture sensor OR an external digital sensor".

**Comment by Sdahl1234** [comment]
Hi @garret 
I have 3 Duka One S6 Plus Wifi and they are all working fine with this integration

**Comment by dingusdk** [comment]
The typo has been fixed in 2024.1.0
The timing may also have been fixed - i was not able to reproduce it, but have made a change: the sleep was not awaited
I will close this issue, if you stille have timing problems when starting, create a new issue.,
